examples/grpc_dynamic_routing/envoy_config.json

```diff
@@ -1044,7 +1044,7 @@
                             "route": {
                               "cluster": "backend-cluster-grpc-echo-oxouww7xzq-uc.a.run.app:443",
                               "hostRewriteLiteral": "grpc-echo-oxouww7xzq-uc.a.run.app",
-                              "idleTimeout": "401s",
```

Comment:
Note this change is only to streaming methods. 

All unary methods still have `400` as response timeout, `401` as stream idle timeout.